applications/OptimizationApplication/python_scripts/utilities/opt_convergence.py

```diff
@@ -0,0 +1,31 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+
+def CreateConvergenceCriteria(parameters: Kratos.Parameters, optimization_problem: OptimizationProblem):
+    type = parameters["type"].GetString()
+    if type == "max_iter":
+        return MaxIterConvCriterium(parameters, optimization_problem)
+    else:
+        raise RuntimeError(f"CreateConvergenceCriteria: unsupported convergence type {type}.")
+
+class MaxIterConvCriterium(object):
```

Comment:
```suggestion
class MaxIterConvCriterium:
```

This is implicitly done in `python3`